packages/office-addin-debugging/package.json

```diff
@@ -40,10 +40,10 @@
     "@types/ws": "^6.0.4",
     "concurrently": "^3.5.0",
     "express": "^4.16.4",
-    "mocha": "^5.2.0",
+    "mocha": "^7.1.1",
     "rimraf": "^2.7.1",
     "ts-node": "^8.5.2",
-    "tslint": "^5.20.1",
+    "tslint": "^6.1.0",
```

Comment:
Another aside, having a work item to move from tslint to eslint. If we can use office-addins-lint for these other packages, which itself is updated to use eslint, that might be useful. Not sure whether this will be difficult. I would hope it would be easy.

Reply:
For now, that's going to have to go on the back-burner.